src/annotation.js

```diff
@@ -108,19 +121,21 @@ const directUpdater = {
   update: Object.assign
 };
 
+function resolveAnnotationOptions(chart, options) {
+  const elType = annotationTypes[options.type] || annotationTypes.line;
+  const mergedOptions = merge(Object.create(null), [chart.options.elements[elType.id], options]);
+  return elType.resolveOptions(chart, mergedOptions);
```

Comment:
So can you just resolve the `display` property here, and no `elType.resolveOptions`.

Reply:
I'm removing them